UnitsNet.Tests/GeneratedCode/IQuantityTests.g.cs

```diff
@@ -252,111 +252,111 @@ public void QuantityInfo_IsSameAsStaticInfoProperty()
         [Fact]
         public void Type_EqualsStaticQuantityTypeProperty()
```

Comment:
Duplicate this Fact method and name it accordingly, in order to keep the old assertion on QuantityType property, and add the new assertion on Info property.

Reply:
done?

Reply:
Maybe I am getting lost in the diffs here, but it seems there are two methods in this test class:
- QuantityInfo_IsSameAsStaticInfoProperty
- Type_EqualsStaticQuantityTypeProperty

I believe this PR incorrectly changed `Type_EqualsStaticQuantityTypeProperty` to be an exact copy of `QuantityInfo_IsSameAsStaticInfoProperty`, they both test the same thing. I think this file should simply be reverted.

Reply:
My bad, I did not see diff for latest commit. Already reverted.